parm/snow/prep/prep_ghcn.yaml.j2

```diff
@@ -0,0 +1,9 @@
+stage:
+  mkdir:
+    - '{{ snow_prepobs_path }}'
+  copy_opt:
+    - ['{{ COMIN_OBS }}/{{ OPREFIX }}ghcn_snow.csv', '{{ DATA }}']
+    - ['{{ FIXgfs }}/gdas/snow/ghcnd-stations.txt', '{{ DATA }}']
```

Comment:
Just a comment, but it'd be great if we could get rid of the ghcnd-stations file. I don't see why the IODA converter should need it.

Reply:
Thanks - I'd forgotten that it has the meta-data. It's unavoidable then.